index.html

```diff
@@ -1,2 +1,43 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link type="text/css" rel="stylesheet" href="css/normalize.css">
+    <link type="text/css" rel="stylesheet" href="css/styles.css">
+    <title>Paintmaster 3000</title>
+</head>
+<body>
+   <div class="container">
+       <h1>Paintmaster 3000</h1>
+       <aside>
+           <div class="button white">
```

Comment:
These should all be on the same line, we're never going to put anything inside them as the div is literally the button